core/spec/models/spree/order_updater_spec.rb

```diff
@@ -197,12 +202,12 @@ module Spree
     it "state change" do
       order.shipment_state = 'shipped'
       state_changes = double
-      allow(order).to receive_messages :state_changes => state_changes
+      allow(order).to receive_messages state_changes: state_changes
       expect(state_changes).to receive(:create).with(
-        :previous_state => nil,
-        :next_state => 'shipped',
-        :name => 'shipment',
-        :user_id => nil
+        previous_state: nil,
+        next_state: 'shipped',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.